src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.scss

```diff
@@ -0,0 +1,52 @@
+@import "scss/constants.scss";
+@import "../Schedule/Schedule.constants.scss";
+
+$image-size: 92px;
+
+$main--margin-top: 2rem;
+$main--margin-bottom: 1rem;
+
+$name--margin-bottom: 5px;
+
+.ScheduleVenueDescription {
+  flex: 1;
+  padding-left: $Schedule--padding;
+
+  &__main {
+    display: flex;
+    align-items: center;
+    margin-top: $main--margin-top;
+    margin-bottom: $main--margin-bottom;
+  }
+
+  &__pic {
+    flex-shrink: 0;
+    width: $image-size;
+    height: $image-size;
+    background-size: cover;
+    border-radius: 50%;
+    margin-right: $Schedule--padding;
+    cursor: pointer;
+    transition: all 400ms $transition-function;
+    box-shadow: 0 8px 16px rgba($black, 0.3);
+    background-image: var(--venue-picture--background-image);
+
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+
+  &__desc {
+    font-size: 0.8rem;
```

Comment:
[NB] use from constants

Reply:
See my comments in https://github.com/sparkletown/sparkle/pull/1302#discussion_r633945091